int32 id
string alias
geometry_msgs/PoseStamped pose
spatial_db/ObjectDescription description
